spyder/widgets/ipythonconsole/client.py

```diff
@@ -232,10 +232,7 @@ def configure_shellwidget(self, give_focus=True):
         self.shellwidget.executed.connect(self.shellwidget.get_cwd)
 
         # To apply style
-        if not create_qss_style(self.shellwidget.syntax_style)[1]:
-            self.shellwidget.silent_execute("%colors linux")
-        else:
-            self.shellwidget.silent_execute("%colors lightbg")
+        self.set_console_scheme(self.shellwidget)

```

Comment:
Please change `set_console_scheme` to `set_color_scheme`

Reply:
There is already a different method called `set_color_scheme`.  Before submitting the PR, I wasn't sure if I should use it or not because it takes `color_scheme` as a parameter, which is a value from the plugin (ipythonconsole.py) and not the widget (client.py).  It was a more complicated change, so I opted for directly fixing this one issue.

Do you want me to redo it using the existing `set_color_scheme`?

Reply:
Yes, please. I think it'd be easier to understand that way. Thanks!

Reply:
OK, will work on it soon.